### DISCUSSION TITLE
Transition from Azure Media Services to MediaKind(MK.IO)

### DISCUSSION THREAD

**Discussion by pallakartheekreddy** [Sunbird-Knowlg/Community#211] [Announcements]
Azure Media Services will be retired on 30 June 2024. Please find more details at the following link:
https://learn.microsoft.com/en-us/azure/media-services/latest/azure-media-services-retirement

To ensure a smooth transition, we plan to use MediaKind, a Microsoft partner solution, as an alternative to Azure Media Services. Our plan includes:

1. Migrating all existing streaming assets to MediaKind.
2. Generating streaming URLs for new assets using MediaKind.

We believe MediaKind will be a suitable replacement for Azure Media Services and will meet our streaming needs effectively. Please find more details about MediaKind in at the following [link](https://mk.io/) 

We welcome any questions or discussions regarding this transition. Please share your thoughts and concerns below.

cc: @vrayulu @madhucr @maheshkumargangula @gandham-santhosh @mohitga @vinukumar-vs

**Comment by mohitga** [comment]
@vinukumar-vs  
This need to taken in the Sunbird Ed and cokreat BB. 
Please have a look and see what is required to adopt the same. 

Please note
 [@Prateek-slokam](https://github.com/Prateek-slokam) [@vinukumar-vs](https://github.com/vinukumar-vs) [@kiranharidas187](https://github.com/kiranharidas187) [@baljeetsingh](https://github.com/baljeetsingh) @[siddhishinde0723](https://github.com/siddhishinde0723) [@parthlawate](https://ekstep.slack.com/team/U0HB3HJ73) @[sanjeev@aastrika.org](mailto:sanjeev@aastrika.org) [@aks30](https://github.com/aks30) [@AmiableAnil](https://github.com/AmiableAnil) [@manojLondhe](https://github.com/manojLondhe) [@swayangjit](https://github.com/swayangjit) [@vinod303](https://github.com/vinod303) [@gandham-santhosh](https://github.com/gandham-santhosh)

**Comment by pallakartheekreddy** [comment]
We are pleased to announce a hot-fix addressing the upcoming retirement of Azure Media Services (AMS) and the transition to MediaKind for our video streaming job.

**Hot-Fix Release Notes**

- Release Notes: [Hot-Fix 6.1.0 (18-06-2024)](https://knowlg.sunbird.org/use/release-notes/release-6.1.0-latest#hot-fix-6.1.0-18-06-2024)
- Detailed release notes for this hot-fix can be found [here](https://knowlg.sunbird.org/use/release-notes/release-6.1.0-latest#hot-fix-6.1.0-18-06-2024). This includes information on migrating from Azure Media Services to MediaKind, new configurations introduced, and steps to apply the update.

**Code Changes**

1. Knowledge Platform Jobs: [Pull Request #795](https://github.com/Sunbird-Knowlg/knowledge-platform-jobs/pull/795)
2. Sunbird Learning Platform: [Pull Request #1989](https://github.com/Sunbird-Knowlg/sunbird-learning-platform/pull/1989)

@vrayulu @madhucr @maheshkumargangula @gandham-santhosh @mohitga @vinukumar-vs

**Reply by vinukumar-vs** [reply]
@pallakartheekreddy as discussed, update the below 2 points in the release notes
* What happens to existing ecars?  - We have to republish all the video contents
* Old env variable is not mentioned in the release notes. 

@mohitga [@gandham-santhosh](https://github.com/gandham-santhosh) [@ashokreddy1208](https://github.com/ashokreddy1208) @vrayulu @madhucr @maheshkumargangula

**Reply by pallakartheekreddy** [reply]
@vinukumar-vs Release notes updated with above details.
